tests/test_losses.py

```diff
@@ -124,3 +125,50 @@ def test_np_loss():
     # y_true, y_preds
     loss = pnl(y_true, y_preds)
     assert loss > 0.9
+
+
+# [soft neasrest neighbor loss]
+def test_softnn_loss_serialization():
+    loss = SoftNearestNeighborLoss()
+    config = loss.get_config()
+    print(config)
+    loss2 = SoftNearestNeighborLoss.from_config(config)
+    assert loss.name == loss2.name
+    assert loss.distance == loss2.distance
```

Comment:
I think this is only loading the distance from default kwarg value and not from the config. I'm not sure if this will pass if we set the distance on line 132 to something else like 'cosine'. 

I think subclassing from MetricLoss should fix this as we add all kwargs to the config.